src/lib/OpenEXRCore/internal_util.h

```diff
@@ -31,7 +31,7 @@ compute_sampled_lines (int height, int y_sampling, int start_y)
         else
             start = start_y;
         end = start_y + height - 1;
-        end -= (end % y_sampling);
+        end -= (end < 0) ? (-end % y_sampling) : (end % y_sampling);
```

Comment:
should we have bailed out before this if someone was asking for a start_y below the height?

Reply:
oh, I am doing math to avoid a loop to find how many samples there are in that range (the c++ does a loop), i just didn't handle when the window is all in the negative correctly and was having an off-by-one error in the chunk size - definitely an exceptional case, but certainly can be encountered if you have a 4:2:2 file with overscan (which one of the c++ tests has, so yay for tests!)...